<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="TechMs-MacBook-Pro.local" failures="1" tests="1" name="Default test" time="12.744" errors="0" timestamp="2022-03-17T11:50:46 PDT">
  <testcase classname="Tests.ShopTests" name="sortByPrice" time="6.199">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;50.0&amp;gt; but was:&amp;lt;120.0&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<50.0> but was:<120.0>
at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:468)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:98)
at Tests.ShopTests.sortByPrice(ShopTests.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- sortByPrice -->
  <testcase classname="Tests.ShopTests" name="nineItemsPerPage" time="0.0">
    <ignored/>
  </testcase> <!-- nineItemsPerPage -->
  <testcase classname="Tests.ShopTests" name="searchErrorMessage" time="0.0">
    <ignored/>
  </testcase> <!-- searchErrorMessage -->
  <testcase classname="Tests.ShopTests" name="searchProduct" time="0.0">
    <ignored/>
  </testcase> <!-- searchProduct -->
  <testcase classname="Tests.ShopTests" name="productCategories" time="0.0">
    <ignored/>
  </testcase> <!-- productCategories -->
  <testcase classname="Tests.ShopTests" name="recentViewedProducts" time="0.0">
    <ignored/>
  </testcase> <!-- recentViewedProducts -->
  <testcase classname="Tests.ShopTests" name="categoryClothes" time="0.0">
    <ignored/>
  </testcase> <!-- categoryClothes -->
</testsuite> <!-- Default test -->
